instance_ips.go

```diff
@@ -19,6 +19,7 @@ type InstanceIPv4Response struct {
 	Private  []*InstanceIP `json:"private"`
 	Shared   []*InstanceIP `json:"shared"`
 	Reserved []*InstanceIP `json:"reserved"`
+	VPC      []*VPCIP      `json:"vpc"`
```

Comment:
It's interesting that these are defined as slices of pointers 🤔 , although that's not the fault of this PR so this should be fine

Reply:
I am not sure.. API returns empty list `[]` instead of `null` when there is not an IP in the category. But yea, I think we can keep the pattern just to be safe.